### ISSUE TITLE
Have you ever encountered evaluation problems？

### ISSUE THREAD

**Issue by zhufeng888** [shuto-keio/3DSSD-TFv2#1]
When I run python lib/core/evaluator.py, the loaded model work well but the compoiled evaluation.cpp always gives me errors.
I also use ubuntu18.04+ cuda10.1 + anaconda3+ python3+ tensorflow-gpu2.3.0. Do you have any ideas? Thank you very much.
 
**** Evluation New Result ****
Assign From checkpoint: /home/hby/hdd/chenyanbin/3DSSD-TFv2/lib/core/log/2021-01-28 16:45:04.066250/model/model-75051
WARNING:tensorflow:From /home/hby/.conda/envs/3DSSD-TFv2/lib/python3.6/site-packages/tensorflow/python/training/tracking/util.py:1298: NameBasedSaverStatus.__init__ (from tensorflow.python.training.tracking.util) is deprecated and will be removed in a future version.
Instructions for updating:
Restoring a name-based tf.train.Saver checkpoint using the object-based restore API. This mode uses global names to match variables, and so is somewhat fragile. It also adds new restore ops to the graph each time it is called when graph building. Prefer re-encoding training checkpoints in the object-based format: run save() on the object-based saver (the same one this message is coming from) and use that checkpoint in the future.
  0%|                                                     | 0/5 [00:00<?, ?it/s]2021-01-29 10:53:57.549457: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcublas.so.10
2021-01-29 10:53:57.653959: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudnn.so.7
100%|█████████████████████████████████████████████| 5/5 [00:01<00:00,  4.37it/s]
loadGroundtruth: no error!
loadDetections: no error!
python: evaluate.cpp:842: bool eval_class(FILE*, FILE*, CLASSES, const std::vector<std::vector<tGroundtruth> >&, const std::vector<std::vector<tDetection> >&, bool, bool, double (*)(tDetection, tGroundtruth, int32_t), std::vector<double>&, std::vector<double>&, std::vector<double>&, DIFFICULTY, METRIC): Assertion `groundtruth.size() == detections.size()' failed.
已放弃 (核心已转储)

**Comment by shorie000** [comment]
Now I tested and it works.
The same problem has been reported in original repository, so you might want to ask there.
https://github.com/Jia-Research-Lab/3DSSD/issues/32

Or you can try running it in a docker environment.
If you're using ubuntu, It shouldn't be too hard.

**Comment by zhufeng888** [comment]
> Now I tested and it works.
> The same problem has been reported in original repository, so you might want to ask there.
> [Jia-Research-Lab/3DSSD#32](https://github.com/Jia-Research-Lab/3DSSD/issues/32)
> 
> Or you can try running it in a docker environment.
> If you're using ubuntu, It shouldn't be too hard.
Thanks, I have solved it. The reason is that I read the wrong path of label_2.
